tests/unit/utils/test_zarr_methods.py

```diff
@@ -0,0 +1,413 @@
+            preprocess=md.preprocess_kerchunk,
+        )
+
+        outfile = tmp_path / "datasets" / "merged_zarr_jsons" / "DummyManager_zarr.json"
+        mzz.return_value.translate.assert_called_once_with(filename=outfile)
+
+    @staticmethod
+    def test_create_zarr_json_w_outfile_path(manager_class, tmp_path, mocker, input_files):
+        mzz = mocker.patch("gridded_etl_tools.utils.zarr_methods.MultiZarrToZarr")
+        md = manager_class()
+        md._root_directory = tmp_path
+
+        md.create_zarr_json(outfile_path="put/it/here")
+
+        mzz.assert_called_once_with(
+            path=list(map(str, input_files.keys())),
+            indicts=list(input_files.values()),
+            remote_protocol="handshake",
+            remote_options={"anon": True},
+            identical_dims=["x", "y"],
+            concat_dims=["z", "zz"],
+            preprocess=md.preprocess_kerchunk,
+        )
+
+        mzz.return_value.translate.assert_called_once_with(filename="put/it/here")
+
+    @staticmethod
+    def test_create_zarr_json_already_done(manager_class, tmp_path, mocker):
+        folder = tmp_path / "datasets" / "merged_zarr_jsons"
+        folder.mkdir(parents=True)
+        outfile = folder / "DummyManager_zarr.json"
+        open(outfile, "w").write("already done")
+
+        mzz = mocker.patch("gridded_etl_tools.utils.zarr_methods.MultiZarrToZarr")
+        md = manager_class()
+        md._root_directory = tmp_path
+
+        md.create_zarr_json(force_overwrite=False)
+        mzz.assert_not_called()
+
+    @staticmethod
+    def test_create_zarr_json_force_overwrite(manager_class, tmp_path, mocker, input_files):
+        folder = tmp_path / "datasets" / "merged_zarr_jsons"
+        folder.mkdir(parents=True)
+        outfile = folder / "DummyManager_zarr.json"
+        open(outfile, "w").write("already done")
+
+        mzz = mocker.patch("gridded_etl_tools.utils.zarr_methods.MultiZarrToZarr")
+        md = manager_class()
+        md._root_directory = tmp_path
+
+        md.create_zarr_json()
+
+        mzz.assert_called_once_with(
+            path=list(map(str, input_files.keys())),
+            indicts=list(input_files.values()),
+            remote_protocol="handshake",
+            remote_options={"anon": True},
+            identical_dims=["x", "y"],
+            concat_dims=["z", "zz"],
+            preprocess=md.preprocess_kerchunk,
+        )
+
+        outfile = tmp_path / "datasets" / "merged_zarr_jsons" / "DummyManager_zarr.json"
+        mzz.return_value.translate.assert_called_once_with(filename=outfile)
+
+    @staticmethod
+    def test_kerchunkify_local(manager_class):
+        md = manager_class()
+        md.local_kerchunk = mock.Mock()
+        md.remote_kerchunk = mock.Mock()
+        md.zarr_json_in_memory_to_file = mock.Mock()
+        assert md.use_local_zarr_jsons is False
+
+        assert md.kerchunkify("/home/body/over/here.json") is md.local_kerchunk.return_value
```

Comment:
Be that as it may, the point of this test is just to see that the output of the dependent function is the value returned by local_kerchunk. Utlimately the value actually returned is determined by an external library, so isn't critical for a unit test.